crates/ursa-tracker/src/types.rs

```diff
@@ -0,0 +1,79 @@
+use hyper::{client::HttpConnector, Body};
+use libp2p::PeerId;
+use serde_derive::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+pub type Client = hyper::client::Client<HttpConnector, Body>;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct NodeAnnouncement {
```

Comment:
Just a quick reminder on the naming here conflicting with index announcement

Reply:
done :smiley: